bundles/specmate-persistency-cdo/src/com/specmate/persistency/cdo/internal/CDOPersistencyService.java

```diff
@@ -147,14 +147,7 @@ public void modified(Config config) throws SpecmateException {
 	@Override
 	public synchronized void start() throws SpecmateException {
 		if (migrationService.needsMigration()) {
-			logService.log(LogService.LOG_INFO, "Data migration needed.");
-			if (migrationService.doMigration()) {
-				// Successful migration
-				logService.log(LogService.LOG_INFO, "Data migration successful.");
-			} else {
-				logService.log(LogService.LOG_ERROR, "Data migration failed.");
-				throw new SpecmateException("Data migration failed.");
-			}
+			migrationService.doMigration();

```

Comment:
what if the migration fails, will the persistency be started? It probably shouldn't be started right?